docusaurus/docs/recipes/video-player.md

```diff
@@ -29,33 +29,37 @@ const Home = () => (
 );
 ```
 
-### 3. Usage
+### 3. Common use cases
 
 Now let's go through some common use cases that might be helpful.
 
-> ⚠️ By default `react-player` sets a width of 640px and a height of 360px to the player. You should change it in order to fit your needs.
+> ⚠️ By default [`react-player`](https://github.com/CookPete/react-player) sets a width of 640px and a height of 360px to the video player. You should change it using [`width` and `height` props](https://github.com/CookPete/react-player#props) in order to make it [responsive](https://github.com/CookPete/react-player#responsive-player) according to your needs.
 
 #### 1. Autoplay
 
-To enable autoplay you need to use `playing` prop.
+To enable autoplay you need to use the [`playing` prop](https://github.com/CookPete/react-player#props), that lets you control the playing of the video. It can be particularly useful when you have a loading state and you only want the video to play after the loading is over.
 
 ```js
-<ReactPlayer url="https://www.video-url.com" playing />
+const [isPlaying, setIsPlaying] = useState(true);
+
+// ...
+
+<ReactPlayer url="https://www.video-url.com" playing={ isPlaying } />
 ```
 
-Be sure to check the different browser policies for autoplaying videos as they are different and you might need additional configuration. For example on Chrome you might need to mute the video in order to autoplay it.
+ℹ️ Be sure to check the different browser policies for [autoplaying videos](https://github.com/CookPete/react-player#autoplay) as they are different and you might need additional configuration. For example, on Chrome you might need to mute the video in order to autoplay it, or in Safari you might need additional work to make it autoplay on low power devices which is a common problem.
```

Comment:
Just a small nitpick, it's not exactly low power devices, but `low power mode`, which is a battery saving mode in iPhones. 

Otherwise, lgtm 👍